Makefile.toml

```diff
@@ -654,6 +670,14 @@ if [[ $ENABLE_COREDUMP == "true" ]]; then
   ulimit -c unlimited
 fi
 
+if [[ -z "$RW_QUERY_LOG_PATH" ]]; then
+  export RW_QUERY_LOG_PATH="${PREFIX_LOG}"
```

Comment:
What about setting this in the `env_script`?

Reply:
What's the difference?

Reply:
Deduplication for `dev` and `playground`. We have to duplicate `ulimit` because it's a shell built-in command that only takes effect within the same script.